analyzers/rspec/cs/Sonar_way_recommended_profile.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If I am correct than this file is not needed.

Reply:
The reason why this file was generated is that the rule is in `Sonar way recommended` profile. I wrote though to @pierre-loup-tristant-sonarsource to confirm that this is intentional.

Reply:
There are a bunch of rules that are part of that profile but this was the first time the file was added. We either do not use that file and do not have to create it, or a lot of rules are missing from them. If the second is the case than we should create a task to add all the other rules that are part of that profile too.

Reply:
I've discussed it in the end with @saberduck and it seems that this was a mistake. The change was needed only for JS. He fixed the RSPEC and I will regenerate the files.